test/integration/default/build/iam.tf

```diff
@@ -87,4 +87,30 @@ output "iam_access_key_recall_hit" {
 
 output "iam_access_key_recall_miss" { 
   value = "AKIAFAKEFAKEFAKEFAKE"
-}
\ No newline at end of file
+}
+
+#======================================================#
+#                    Flow Logs
```

Comment:
The flow log fixture code should not be in the IAM fixture file.  IAM has to do with users and permissions.  Flow logs are about networking; it should be in either ec2.tf, or in a new vpc.tf file.